views/js/runner/plugins/content/accessibility/keyNavigation.js

```diff
@@ -379,43 +440,72 @@ define([
      * @param testRunner
      * @returns {*}
      */
-    function initTestRunnerNavigation(testRunner){
-
+    function initTestRunnerNavigation(testRunner, state){
```

Comment:
Should not it be `config` instead of `state`?

Reply:
fixed